### DISCUSSION TITLE
Show more than one metric from the same dataset?

### DISCUSSION THREAD

**Discussion by joshstrike** [chartbrew/chartbrew#222] [General]
Hey - I'm probably stupid, I spent hours setting up a chartbrew under nginx to see if I could run it that way and look at a big dataset. I'm in now, and I'm trying to run a query like this:

`SELECT COUNT(customerID), franchiseID, DATE(created) AS dc 
FROM fra_cus 
 WHERE created > '2016-01-01'
GROUP BY franchiseID, DATE(created);`

Wherein I want to see separate metrics on the chart based on the GROUP. Essentially what I see is that I can change a filter and show customers from each franchise one by one -- what I'm wondering is, is there no way to show *more than one line on a chart inside a dataset*? i.e.. split the data based on a second parameter so you can see one against the other? Do I have to create a separate dataset/query for every single WHERE parameter and then unify them later somehow?

**Comment by razvanilin** [comment]
Hey @joshstrike in Chartbrew you have to create new datasets for each line you want to see on your chart. The datasets will then share the X&Y axes and display on the chart at the same time

Let me know if you give this a try. Curious to see if this works for your use case

It will also be helpful if you can add an image with a visualization that you have in mind. Maybe it will make it easier to understand
